code/game/objects/structures/morgue.dm

```diff
@@ -268,13 +268,19 @@ GLOBAL_LIST_EMPTY(crematoriums)
 			else
 				M.log_message("was cremated", LOG_ATTACK)
 
+			for(var/obj/item/I in M)
+				if(I.resistance_flags & INDESTRUCTIBLE)
+					M.dropItemToGround(I, TRUE)
+
 			M.death(1)
 			if(M) //some animals get automatically deleted on death.
 				M.ghostize()
 				qdel(M)
 
 		for(var/obj/O in conts) //conts defined above, ignores crematorium and tray
 			CHECK_TICK
+			if(O.resistance_flags & INDESTRUCTIBLE)
```

Comment:
```suggestion
			if(O.resistance_flags & INDESTRUCTIBLE)
				O.forceMove(src) // in case an item in container should be spared
```

This way every `INDESTRUCTIBLE` item can survive and actually show up.